catalog.html

```diff
@@ -56,23 +56,201 @@
+                    <div class="faq__answer">
+                        <label class="form-radio form-radio--col">
+                            <input class="input-hidden form-radio__input" type="radio" name="yellow" required>
+                            <span class="form-radio__inner">
+                                <span class="form-radio__box"></span>
+                                <span class="form-radio__text">Yellow</span>
+                            </span>
+                        </label>
+                        <label class="form-radio form-radio--col">
+                            <input class="input-hidden form-radio__input" type="radio" name="green" required>
+                            <span class="form-radio__inner">
+                                <span class="form-radio__box"></span>
+                                <span class="form-radio__text">Green</span>
+                            </span>
+                        </label>
+                        <label class="form-radio form-radio--col">
+                            <input class="input-hidden form-radio__input" type="radio" name="mix" required>
+                            <span class="form-radio__inner">
+                                <span class="form-radio__box"></span>
+                                <span class="form-radio__text">Mix</span>
+                            </span>
+                        </label>
+                        <label class="form-radio form-radio--col">
+                            <input class="input-hidden form-radio__input" type="radio" name="orange" required>
+                            <span class="form-radio__inner">
+                                <span class="form-radio__box"></span>
+                                <span class="form-radio__text">Orange</span>
+                            </span>
+                        </label>
+                    </div>
+                </details>
+                <details class="faq__item">
+                    <summary class="faq__question">Face</summary>
+                        <div class="faq__answer">
+                        </div>
+                </details>
+                <details class="faq__item">
+                    <summary class="faq__question">Skin</summary>
+                        <div class="faq__answer">
+                        </div>
+                </details>
+                <details class="faq__item">
+                    <summary class="faq__question">Background</summary>
+                        <div class="faq__answer">
+                        </div>
+                </details>
+                <details class="faq__item">
+                    <summary class="faq__question">Special</summary>
+                    <div class="faq__answer">
+                        <label class="form-radio form-radio--col">
+                            <input class="input-hidden form-radio__input" type="radio" name="yellow" required>
```

Comment:
According to design the checkboxes should be here.

Reply:
done